tests/cases/fourslash/codeFixMissingTypeAnnotationOnExports54-generator-generics.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path='fourslash.ts'/>
+
+// @isolatedDeclarations: true
+// @declaration: true
+// @lib: es2015
+//// export function foo(x: Generator<number>) {
+////     return x;
+//// }
+
+verify.codeFix({
+    description: "Add return type 'Generator<number>'",
+    index: 0,
+    newFileContent:
+`export function foo(x: Generator<number>): Generator<number> {
```

Comment:
To be clear I was meaning to write `function *foo() { }` here and then see what happens.

Reply:
Ah, OK, added.

It's not technically related to the rest of this PR since it gets inferred a precise return that needs all 3 type arguments, but it's a good case to include in the test suite anyway.